tests/data/test.nq

```diff
@@ -0,0 +1,6 @@
+<http://example.org/stuff/1.0/document> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://xmlns.com/foaf/0.1/Document> <http://example.com/graph1> .
+<http://example.org/stuff/1.0/document> <http://xmlns.com/foaf/0.1/creator> <http://example.org/stuff/1.0/creator> <http://example.com/graph1> .
+<http://example.org/stuff/1.0/document> <http://xmlns.com/foaf/0.1/title> "Example Document" <http://example.com/graph2> .
+<http://example.org/stuff/1.0/creator> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://xmlns.com/foaf/0.1/Person> <http://example.com/graph3> .
+<http://example.org/stuff/1.0/creator> <http://xmlns.com/foaf/0.1/name> "John Doe" <urn:x-rdflib:default> .
+<http://example.org/stuff/1.0/creator> <http://xmlns.com/foaf/0.1/mbox> <mailto:johndoe@example.org> <urn:x-rdflib:default> .
```

Comment:
It would be great to test with the actual NQuads default graph like  `<http://example.org/stuff/1.0/creator> <http://xmlns.com/foaf/0.1/mbox> <mailto:johndoe@example.org> .`.

I believe proper support for the default graph would require moving the quads in a named graph (likely `urn:x-rdflib:default`). Sadly pyoxigraph 0.3.x does not support that but pyoxigraph 0.4 alphas does.